content/engine/release-notes/24.0.md

```diff
@@ -20,6 +20,36 @@ For more information about:
 - Deprecated and removed features, see [Deprecated Engine Features](../deprecated.md).
 - Changes to the Engine API, see [Engine API version history](../api/version-history.md).
 
+## 24.0.7
+
+{{< release-date date="2023-10-26" >}}
+
+For a full list of pull requests and changes in this release, refer to the relevant GitHub milestones:
+
+- [docker/cli, 24.0.7 milestone](https://github.com/docker/cli/issues?q=is%3Aclosed+milestone%3A24.0.7)
+- [moby/moby, 24.0.7 milestone](https://github.com/moby/moby/issues?q=is%3Aclosed+milestone%3A24.0.7)
+
+### Bug fixes and enhancements
+
+* Ensure that overlay2 layer metadata is correct. [moby/moby#46703](https://github.com/moby/moby/pull/46703)
+* Fix "Rootful-in-Rootless" Docker-in-Docker on systemd versions 250 and later. [moby/moby#46626](https://github.com/moby/moby/pull/46626)
+* Fix `docker.socket` not getting disabled when uninstalling the `docker-ce` RPM package. [docker/docker-ce-packaging#852](https://github.com/docker/docker-ce-packaging/pull/852)
+* Fix `dockerd-rootless-setuptools.sh` when username contains a backslash. [moby/moby#46407](https://github.com/moby/moby/pull/46407)
+* Fix a bug that would prevent network sandboxes to be fully deleted when stopping containers with no network attachments and when `dockerd --bridge=none` is used. [moby/moby#46702](https://github.com/moby/moby/pull/46702)
+* Fix a bug where context cancellation could interrupt container restart. [moby/moby#46697](https://github.com/moby/moby/pull/46697)
+* Fix an issue where libnetwork would accept a ChildSubnet with a bigger mask than its parent subnet. [moby/moby#46482](https://github.com/moby/moby/pull/46482)
```

Comment:
Are these two summaries ok? wasn't sure what to put here

Reply:
The linked issue was https://github.com/docker/for-mac/issues/6870

So it looks like; failing to start a container when providing a ` --ip-range` with a mask that is larger than its parent subnet (or something along those lines? I'm not good at this either 🙈)